config.json

```diff
@@ -28,6 +28,20 @@
         "strings"
       ]
     },
+    {
+      "slug": "high-scores",
+      "uuid": "574d6323-5ff5-4019-9ebe-0067daafba13",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 1,
```

Comment:
This is probably difficulty 2 since it deals with storing member variables and then answering questions about those member variables.

NOTE: you should also move this down in the file into the difficulty 2 section (after the core exercise).